### PR TITLE
feat: make demo site accessible directly from navbar

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

## Description
* AS-IS : demo site is accessible through following steps: "Demo" in navbar  -> "See Datahub In Action " docs -> "Check out the demo" button  -> actual demo site 
* TO-BE :  "Demo" in navbar -> actual demo site 
* keep the "See Datahub In Action " docs just in case for future use cases. 

Any ideas are welcome regarding this change!

resolves DR-379

### CODE REVIEW

File: docs-website/docusaurus.config.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ module.exports = {
           position: "right",
         },
         {
-          to: "docs/demo",
```
Comment (hsheth2) [inline docs-website/docusaurus.config.js:57]: can we also delete the demo.md page?

**Review by hsheth2 — COMMENTED** [review]
do we also need update sidebar.js?

File: docs-website/docusaurus.config.js (hunk 1 above)
Comment (yoonhyejin) [reply docs-website/docusaurus.config.js:57]: hmm okay, i was about to leave it for future use cases but tbh I think it would be find without it.

**Review by hsheth2 — COMMENTED** [review]
Can we actually keep a link to demo in the sidebar as well, but make it directly open the demo environment

You can do that like this: https://docusaurus.io/docs/sidebar/items#sidebar-item-link

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**